### ISSUE TITLE
Cover art display

### ISSUE THREAD

**Issue by cascardian** [m64p/mupen64plus-gui#19]
This is predicated on https://github.com/m64p/mupen64plus-gui/issues/18.

A more frontend-like feature might be a cover art box icon besides the game name (Dolphin banner-style) and/or a separate box art view (like RPCS3), both of which could be made resizable. The cover art could be pulled via the scraping APIs of [TheGamesDB](http://wiki.thegamesdb.net/index.php/API_Introduction) (free but can sometimes be slow) or [MobyGames](https://www.mobygames.com/info/api)/[GiantBomb](https://www.giantbomb.com/forums/api-developers-3017/quick-start-guide-to-using-the-api-1427959/) (requires an account for the API key).

This is also something the [Mupen64Plus-Qt launcher](https://github.com/dh4/mupen64plus-qt) features, so maybe you could pull some code/get some inspiration from there. In general, that frontend has some QoL features that deserve a look if these sorts of things are within the scope of your project.

**Comment by loganmc10** [comment]
I'll be honest, I never really got the appeal of these kinds of features. If someone else wants to implement this sort of thing I'll merge it, but I doubt I'll work on it myself. Programs like LaunchBox exist if people want that sort of pretty interface already

**Comment by scottluker** [comment]
Yeah I personally use LaunchBox for a pretty interface and prefer emulator guis to just be functional and simplistic.
